src/modules/dashboard/components/Dialogs/GnosisControlSafeDialog/SafeTransactionPreview.tsx

```diff
@@ -0,0 +1,205 @@
+import React, { useMemo } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import { DialogSection } from '~core/Dialog';
+import { Annotations, Input } from '~core/Fields';
+import Heading from '~core/Heading';
+import Numeral from '~core/Numeral';
+import TokenIcon from '~dashboard/HookedTokenIcon';
+
+import { Colony, AnyUser } from '~data/index';
+
+import AddressDetailsView from './TransactionPreview/AddressDetailsView';
+import { FormValues } from './GnosisControlSafeDialog';
+import { TransactionTypes, MSG as ConstantsMSG } from './constants';
+import DetailsItem from './DetailsItem';
+import styles from './GnosisControlSafeForm.css';
+
+const MSG = defineMessages({
+  previewTitle: {
+    id: 'dashboard.GnosisControlSafeDialog.SafeTransactionPreview.previewTitle',
+    defaultMessage: 'Confirm transaction details',
+  },
+  transactionsSetTitle: {
+    id: `dashboard.GnosisControlSafeDialog.SafeTransactionPreview.transactionsSetTitle`,
+    defaultMessage: 'Title of the set of all the arbitrary transactions',
+  },
+  explainWhy: {
+    id: 'dashboard.GnosisControlSafeDialog.SafeTransactionPreview.explainWhy',
+    defaultMessage: `Explain why you’re making arbitrary transaction (optional)`,
```

Comment:
I think it would sound better if we add the word "the": "Explain why you’re making the arbitrary transaction (optional)"

Reply:
Never mind, I was going through your code and while I was reviewing it you already made the changes.